src/test/java/tests/DataTables.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import constants.Urls;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DataTables extends BaseTest {
+    @Test
+    public void nameCheckTest() {
+        //Open url of page
+        driver.get(Urls.DATA_TABLES_URL);
+        //Add check
+        driver.findElement(By.xpath("//table[@id='table1']//td[text()='fbach@yahoo.com']"));
+        WebElement elementLastName = driver.findElement(By.xpath("//table[@id='table1']//td[text()='fbach@yahoo.com']//preceding-sibling::td[contains(text(),'Bach')]"));
+        String lastNameActual = elementLastName.getText();
```

Comment:
Please, work a little bit with locators in tables, because here we expect that name and lastname can change, here you can use access by index, i.e. (//table[@id='table1']//td[text()='fbach@yahoo.com']//preceding-sibling::td)[n]